examples/update-on-async-rendering/definition-getsnapshotbeforeupdate.js

```diff
@@ -0,0 +1,9 @@
+class Example extends React.Component {
+  getSnapshotBeforeUpdate(prevProps, prevState) {
+    // ...
```

Comment:
Maybe return snapshot here?

Reply:
Then it'll be a fully fledged example. Maybe I should just remove `componentDidUpdate`? There's a full example below anyway.

Reply:
I'll remove cDU to focus this on the new method signature.